app/views/match_decisions/_record_voucher_date_housing_subsidy_admin.haml

```diff
@@ -0,0 +1,25 @@
+.match-decision.c-card.c-card--flush.card--block
+  .c-card__content
+    = simple_form_for @decision, url: access_context.match_decision_path(@match, @decision) do |form|
+      .o-choose.o-choose--flush.o-choose--flush-top
+        - if @decision.accessible_by?(current_contact)
+          .o-choose__choice{class: [('o-choose__choice--disabled' if !@decision.editable?)]}
+            %header
+              .o-choose__title
+                .c-choice-icon.c-choice-icon--confirm.mr-4
+                %h3 Continue Match
+            .o-choose__content
+              %p
+                Please record the date the voucher was issued.
+              = form.input :date_voucher_issued, label: 'Date voucher issued', as: :date_picker, disabled: !@decision.editable?
```

Comment:
maybe?
```suggestion
              = form.input :date_voucher_issued, label: 'Date voucher issued', as: :date_picker, disabled: !@decision.editable?, required: true
```

Reply:
We don't seem to have done this elsewhere in CAS -- perhaps because the field can be blank if the match is declined or canceled?